AeroGearPush-Sw.podspec

```diff
@@ -1,11 +1,11 @@
 Pod::Spec.new do |s|
   s.name         = "AeroGearPush-Sw"
-  s.version      = "3.0.7"
+  s.version      = "3.0.1"
   s.summary      = "AeroGear UnifiedPush Client Registration SDK (Swift)."
-  s.homepage     = "https://github.com/coco-jam/aerogear-ios-push"
+  s.homepage     = "https://github.com/aerogear/aerogear-ios-push/tree/Swift4.2"
```

Comment:
You don't need to add a branch here. It's not used to install the lib, It's only to shows the aerogear website on cocoapods website.

Reply:
As we are not pushing to cocoapods suppose it doesn't matter what here. Will change